systemtest/src/main/java/io/strimzi/systemtest/keycloak/KeycloakInstance.java

```diff
@@ -107,28 +76,15 @@ public String getNamespace() {
     public String getHttpsUri() {
         return httpsUri;
     }
-    public String getHttpUri() {
-        return httpUri;
-    }
     public String getValidIssuerUri() {
         return validIssuerUri;
     }
-    public void setValidIssuerUri(String validIssuerUri) {
-        this.validIssuerUri = validIssuerUri;
-    }
     public String getJwksEndpointUri() {
         return jwksEndpointUri;
     }
-    public void setJwksEndpointUri(String jwksEndpointUri) {
-        this.jwksEndpointUri = jwksEndpointUri;
-    }
     public String getOauthTokenEndpointUri() {
         return oauthTokenEndpointUri;
     }
-
-    public void setOauthTokenEndpointUri(String oauthTokenEndpointUri) {
-        this.oauthTokenEndpointUri = oauthTokenEndpointUri;
-    }
```

Comment:
I think this is ok to remove because these parameters are assigned in the construction phase...and when we changing the realms we replacing these values with private attributes. So it's ok to remove.